src/Discord/MessageSender.php

```diff
@@ -52,6 +53,14 @@ public function sendSecretMessage(SecretSanta $secretSanta, string $giver, strin
 
         try {
             $this->apiHelper->sendMessage($giver, $text);
+        } catch (CommandClientException $e) {
+            $precision = null;
+
+            if (($response = $e->getResponse()) && 403 === $response->getStatusCode()) {
+                $precision = 'They do not allow to receive DM. Please ask him to change his server settings.';
```

Comment:
They? Him?

```suggestion
                $precision = 'Do not allow to receive DM from applications. Please ask the user to change his server settings.';
```

Reply:
They is the common form (singular) to do not choose between her / him

so I'm :+1: for this change

Reply:
Oups, indee.
With the concatenation, it will give the following message:
```
Fail to send message to @pyrech. Do not allow to receive DM from applications. Please ask the user to change his server settings.
```
And Discord DM parameter is not only for bot messages AFAIK.

What about:

```
Fail to send message to @pyrech. The user does not allow to receive DM. Please ask them to change their server settings.
```